src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/ProjectItemDetails.xaml.cs

```diff
@@ -20,7 +20,9 @@ public ProjectItemDetails()
 
 		protected override void OnNavigatedTo(NavigationEventArgs e)
 		{
+#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
You could probably disable the warning using the SurpressMessage tag like in ProjectItemDetailsPageViewModel.cs, it's also a bit cleaner like that :)

Reply:
I do agree but it does not seems to remove the warning, can't compile : 

```
[SuppressMessage("Call is not async", "CS4014", Justification = "Because this call is not awaited, execution of the current method continues before the call is completed")]
		protected override void OnNavigatedTo(NavigationEventArgs e)
		{
			(DataContext as ProjectItemDetailsPageViewModel).OnNavigatedTo(e.Parameter as RichWorkItem);
			base.OnNavigatedTo(e);
		}
```

Reply:
Can you add a TODO with the corresponding task item from the backlog if we can't fix this now?